qutebrowser/config/configdata.py

```diff
@@ -1512,12 +1512,12 @@ def data(readonly=False):
         ('yank -ds', ['yD']),
         ('yank -p', ['yp']),
         ('yank -ps', ['yP']),
-        ('paste', ['pp']),
-        ('paste -s', ['pP']),
-        ('paste -t', ['Pp']),
-        ('paste -ts', ['PP']),
-        ('paste -w', ['wp']),
-        ('paste -ws', ['wP']),
+        ('open {clipboard}', ['pp']),
+        ('open {primary}', ['pP']),
+        ('open -t {clipboard}', ['Pp']),
+        ('open -t {primary}', ['PP']),
+        ('open -w {clipboard}', ['wp']),
+        ('open -w {primary}', ['wP']),
```

Comment:
Can you please add migrations for the existing bindings to the bottom of this file, and corresponding tests to `test_migrations` in `tests/unit/config/test_config.py`?